integration-tests/universal/log_poller/helpers.go

```diff
@@ -276,29 +277,30 @@ var emitEvents = func(ctx context.Context, l zerolog.Logger, logEmitter *contrac
 	}
 }
 
-var waitForEndBlockInLogPoller = func(endBlock int64, chainID *big.Int, l zerolog.Logger, coreLogger core_logger.SugaredLogger, nodes *test_env.ClCluster) (bool, error) {
-	for i := 1; i < len(nodes.Nodes); i++ {
-		clNode := nodes.Nodes[i]
-		orm, db, err := NewOrm(coreLogger, chainID, clNode.PostgresDb)
-		if err != nil {
-			return false, err
-		}
-
-		defer db.Close()
-		block, err := orm.SelectBlockByNumber(endBlock)
-		if err != nil {
-			return false, err
-		}
-
-		if block == nil {
-			return false, nil
-		}
-	}
-
-	return true, nil
-}
-
-var chainHasFinalisedEndBlock = func(l zerolog.Logger, evmClient ctf_blockchain.EVMClient, endBlock int64) (bool, error) {
+// Unused???
```

Comment:
Are these planned to be used @Tofel

Reply:
if you mean `waitForEndBlockInLogPoller` then we can delete it, because it doesn't add that much value to know whether a block is in LP if it wasn't finalised yet, and for that I have `logPollerHasFinalisedEndBlock()`.

Reply:
Alright I will drop this code.